ExchangeSharp/API/Exchanges/BL3P/Enums/BL3PCurrencyFee.cs

```diff
@@ -0,0 +1,8 @@
+namespace ExchangeSharp.API.Exchanges.BL3P.Enums
+{
+	public enum BL3PCurrencyFee : byte
+	{
```

Comment:
Should be internal

Reply:
This can't be made `internal`, because, it's one of the settings of `ExchangeBL3PAPI`. See prop `ExchangeBL3PAPI.DefaultFeeCurrency`.